vignettes/trap_effort.Rmd

```diff
@@ -305,17 +250,75 @@ usethis::use_data(weekly_hours_fished, overwrite = TRUE)
 
 ### QC
 
-The following plots provide visuals to show weekly hours fished for all tributaries. Note that this is for all years of data so there are multiple datapoints for a given week due to multiple years being represented.
+The following plots provide visuals to show weekly hours fished for a few select stream and calendar year. 
+Note that for the combined plots this is for all years of data so there are multiple datapoints for a given week due to multiple years being represented and multiple subsites.
 
 ```{r, echo = FALSE}
 weekly_hours_fished %>% 
-  ggplot(aes(x = week, y = hours_fished, color = factor(site))) +
+  filter(stream == "battle creek", year == 2022) |> 
+  ggplot(aes(x = week, y = hours_fished, color = factor(subsite))) +
   geom_point() + 
-  facet_wrap(~stream, scales = "free")+ 
+  facet_wrap(~year, scales = "free")+ 
   scale_color_manual(values = colors_full) +
   theme_minimal() +
       labs(x = "Week", y = "Hours Fished", 
-       title = "Hours Fished All Methods")
+           color = "",
+       title = "Batte Creek Hours Fished")
```

Comment:
* Should be "**Battle** Creek Hours Fished" right now has Batte

Reply:
done